### ISSUE TITLE
E203 is not ignored via pyproject.toml

### ISSUE THREAD

**Issue by RJ722** [hhatto/autopep8#559]
If `E203` (I've not checked with other issues, happy to do so if required) is specified under `ignore` key in `pyproject.toml`, it is not being ignored, and is still fixed. Let me give you a more concrete example:

```python
# mock file - a.py

li = [1, 2, 3, 10]
print(li[1 : 2])
```

`pyproject.toml` at the root of the project:

```toml
[tool.autopep8]
ignore = "E203"
```

When I run, `autopep8 -dvvvv a.py`:

```diff
enable pyproject.toml config: section=tool.autopep8, key=ignore, value="E203"
[file:a.py]
--->  Applying global fix for E265
--->  1 issue(s) to fix {'E203': {4}}
--->  0 issue(s) to fix {}
--- original/a.py
+++ fixed/a.py
@@ -1,4 +1,4 @@
 # mock file

 li = [1, 2, 3, 10]
-print(li[1 : 2])
+print(li[1: 2])
```

As you can see, `E203` isn't ignored. However, if I run `autopep8 -dvvvv --ignore E203`, or I specify `E203` under `pycodestyle` in my `setup.cfg`, it is ignored correctly. Here's the output of `autopep8 -dvvvv --ignore E203`:

```
enable pyproject.toml config: section=tool.autopep8, key=ignore, value="E203"
[file:a.py]
--->  Applying global fix for E265
--->  0 issue(s) to fix {}
```

---

## Your Environment
* Python version: Python 3.8.2
* autopep8 version: `autopep8 1.5.2 (pycodestyle: 2.5.0)`
* Platform: macOSX

I shall be happy to provide more information if required. Thanks for the great project!

**Comment by hhatto** [comment]
@RJ722 

This problem fix in [version 1.5.3](https://github.com/hhatto/autopep8/releases/tag/v1.5.3).
Please try again, after running the `pip install --upgrade autopep8` command.
```
$ cat tt.py
li = [1, 2, 3, 10]
print(li[1 : 2])

$ autopep8 --version
autopep8 1.5.3 (pycodestyle: 2.6.0)

$ autopep8 -d tt.py -vvv
enable pyproject.toml config: key=ignore, value=E203
[file:tt.py]
--->  Applying global fix for E265
--->  0 issue(s) to fix {}
```

Thanks

**Comment by Borda** [comment]
I have a problem with E731
```
enable config: section=flake8, key=select, value=E,W,F
enable config: section=flake8, key=verbose, value=2
enable config: section=flake8, key=ignore, value=
E731  # Do not assign a lambda expression, use a def
enable pyproject.toml config: key=ignore, value=E731
[file:torchmetrics/__about__.py]
--->  Applying global fix for E265
--->  0 issue(s) to fix {}
...
[file:torchmetrics/classification/stat_scores.py]
--->  Applying global fix for E265
--->  1 issue(s) to fix {'E731': {189}}
--->  2 issue(s) to fix {'E301': {189}, 'E704': {189}}
--->  1 issue(s) to fix {'E704': {190}}
...
```
